src/main/java/com/capgemini/fs/coindashboard/CRUDService/queries/GetQueriesImplementation.java

```diff
@@ -45,6 +45,32 @@ public String getCoinMarketData(
     return gson.toJson(result);
   }
 
+  @Override
+  public String getCoinDetails(String symbol) {
+    MatchOperation matchStage = Aggregation.match(new Criteria("symbol").is(symbol));
+    ProjectionOperation projectStage =
+        Aggregation.project(
+                "name",
+                "symbol",
+                "image_url",
+                "genesis_date",
+                "is_token",
+                "contract_address",
+                "links",
+                "description")
+            .andExclude("_id");
+    Aggregation aggregation =
+        Aggregation.newAggregation(matchStage, projectStage, Aggregation.limit(1));
```

Comment:
This limit is confusing and I had a chat with Filip to understand its purpose.

Given the current state of matter:
- I wonder if Mongo gives any guarantee on order of returned results. If not, it may sometime cause an issue. Initially I didn't complain about presence of limit (it may be obviously applicable from the requirement perspective), but rather the fact that you pick one item from a bag without having confidence which one will you get - and in this case I suppose it could be the newest one that should be picked (so some sorting should be in order).
- **If you plan to fix this in future, a JIRA ticket should be created and planned ASAP to not forget about this.**
- Out of two ideas presented in chat, I suppose that updating is better than cleaning out data. Basically, if you plan to have a unique identifier, make sure that system prevents duplicates: https://www.mongodb.com/docs/v5.0/core/index-unique/
- Once you will fix this, please don't leave this limit here if it would be useless. It would just confuse anyone debugging the code and looking for cause in case of any issue.